--- a/vendor/cogpowered/finediff/src/cogpowered/FineDiff/Render/Html.php	2014-02-04 13:25:20.022308605 -0800
+++ b/vendor/cogpowered/finediff/src/cogpowered/FineDiff/Render/Html.php	2014-02-04 13:35:56.786300046 -0800
@@ -25,7 +25,7 @@
     public function callback($opcode, $from, $from_offset, $from_len)
     {
         if ($opcode === 'c') {
-            $html = htmlentities(htmlentities(substr($from, $from_offset, $from_len)));
+            $html = htmlentities(substr($from, $from_offset, $from_len), ENT_COMPAT, 'UTF-8');
         } else if ($opcode === 'd') {
 
             $deletion = substr($from, $from_offset, $from_len);
@@ -34,12 +34,12 @@
                 $deletion = str_replace(array("\n","\r"), array('\n','\r'), $deletion);
             }
 
-            $html = '<del>'.htmlentities(htmlentities($deletion)).'</del>';
+            $html = '<del>'.htmlentities($deletion, ENT_COMPAT, 'UTF-8').'</del>';
 
         } else /* if ( $opcode === 'i' ) */ {
-            $html = '<ins>'.htmlentities(htmlentities(substr($from, $from_offset, $from_len))).'</ins>';
+            $html = '<ins>'.htmlentities(substr($from, $from_offset, $from_len), ENT_COMPAT, 'UTF-8').'</ins>';
         }
 
         return $html;
     }
-}
\ No newline at end of file
+}
